user-interface/7001-DATA-data_display.md

```diff
@@ -0,0 +1,41 @@
+# Data display
+
+This is a definition of some common data types and the rules about the display them. These are referenced in other acceptenace criteria to avoid repetition.
+
+## Size
+
+>aka contracts, volume, amount.
+
+This is set per-market and represent the number of contracts that are being brought or sold.
+
+`Market.positionDecimalPlaces` tells us where to put the decimal when displaying the number. Size can be a whole number if `Market.positionDecimalPlaces` = 0, or a [fractional order](../protocol/0052-FPOS-fractional_orders_positions.md) if > 0.
+It **should** always be displayed to the full number of decimal places. however, there may be exceptions, e.g. when visualizing on a depth chart, where the precision is not required.
+
+## Quote price
+
+> aka. price, quote, level.
+
+This is set per-market and represent the "price" of an asset. It can have a 1-1 relationship with the settlement asset but it is also possible that products will have different payoff methods, which is one of the reasons we don't just use settlement asset (another being in the future some markets could have multiple settlement assets).
```

Comment:
In general prices will have less decimals than settlement assets because noone wants 18dp prices on order books :)